docs/kind-ipv6-dualstack.md

```diff
@@ -0,0 +1,280 @@
+sudo mv /usr/bin/kubectl /usr/bin/kubectl-v1.17.3
+sudo ln -s /usr/bin/kubectl-v1.17.3 /usr/bin/kubectl
+```
+
+Download and install latest version of `kubectl`:
+
+```
+$ K8S_VERSION=v1.18.0
+$ curl -LO https://storage.googleapis.com/kubernetes-release/release/$K8S_VERSION/bin/linux/amd64/kubectl
+$ chmod +x kubectl
+$ sudo mv kubectl /usr/bin/kubectl-v1.18.0
+$ sudo rm /usr/bin/kubectl
+$ sudo ln -s /usr/bin/kubectl-v1.18.0 /usr/bin/kubectl
+$ kubectl version --client
+Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-25T14:58:59Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+### Docker Changes For Dual-stack
+
+For dual-stack, IPv6 needs to be enable in Docker on the host same as
+for IPv6 only. See above: [Docker Changes For IPv6](#docker-changes-for-ipv6)
+
+### KIND With IP Dual-stack
+
+IP dual-stack is not currently supported in KIND. There is a PR
+([692](https://github.com/kubernetes-sigs/kind/pull/692))
+with IP dual-stack changes. Currently using this to test with. A pre-built
+binary can be downloaded, or it can be built locally.
+
+Optionally, save previous version of KIND (if it exists):
+
+```
+cp $GOPATH/bin/kind $GOPATH/bin/kind.orig
+```
+
+#### Download Pre-built KIND With Dual-stack
+
+To install the dual-stack patched version of KIND:
+
+```
+wget https://github.com/aojea/kind/releases/download/dualstack/kind.linux
+chmod +x kind.linux
+mv kind.linux $GOPATH/bin/kind
+```
+
+#### Build KIND With Dual-stack Locally
+
+To build locally (if additional needed):
+
+```
+go get github.com/kubernetes-sigs/kind; cd $GOPATH/src/github.com/kubernetes-sigs/kind
+git pull --no-edit --strategy=ours origin pull/692/head
+make clean
+make install INSTALL_DIR=$GOPATH/bin
+```
+
+### OVN-Kubernetes With IP Dual-stack
+
+For status of IP dual-stack in OVN-Kubernetes, see
+[1142](https://github.com/ovn-org/ovn-kubernetes/issues/1142).
+
+To run OVN-Kubernetes with IP dual-stack in a KIND deployment, run:
+
+```
+$ go get github.com/ovn-org/ovn-kubernetes; cd $GOPATH/src/github.com/ovn-org/ovn-kubernetes
+
+$ cd go-controller/
+$ make
+
+$ cd ../dist/images/
+$ make fedora
+
+$ cd ../../contrib/
+$ KIND_IPV4_SUPPORT=true KIND_IPV6_SUPPORT=true K8S_VERSION=v1.18.0 ./kind.sh
+```
+
+Once `kind.sh` completes, setup kube config file:
+
+```
+$ cp ~/admin.conf ~/.kube/config
+-- OR --
+$ KUBECONFIG=~/admin.conf
+```
+
+Once testing is complete, to tear down the KIND deployment:
+
+```
+$ kind delete cluster --name ovn
+```
+
+### Current Status
+
+This is subject to change because code is being updated constantly. But this is
+more a cautionary note that this feature is not completely working at the
+moment.
+
+The nodes do not go to ready because the OVN-Kubernetes hasn't setup the network
```

Comment:
just curious, what's missing to setup the network?

Reply:
ovn-kubernetes currently half-supports and half-doesn't-support dual-stack (#1142). Until that's finished the test job isn't really expected to work.